apps/daimo-mobile/src/view/shared/SearchHeader.tsx

```diff
@@ -86,8 +86,7 @@ export function SearchHeader({
   if (account == null) return null;
   const eAcc: EAccountContact = {
     type: "eAcc",
-    addr: account.address,
-    name: account.name,
+    ...toEAccount(account),
```

Comment:
could be simplified to `const eAcc: EAccountContact = toEAccount(account)` ?

Reply:
`toEAccount` returns an `EAccount`, not an `EAccountContact`. I'll make a `toEAccountContact` function